### PR TITLE
Switch to ffimageloading

### PR DESCRIPTION
**Pull request by matt-goldman**
Add loading placeholder
Use ffimageloading greyscale transform for unscanned devs

### CODE REVIEW

**Comment by matt-goldman** [comment]
DON'T MERGE THIS until this has been merged first: https://github.com/SSWConsulting/SSW.Rewards/pull/386

**Review by bradystroud — APPROVED** [review]

**Pull request merged**
